R/derive_param_computed.R

```diff
@@ -194,6 +200,24 @@
 #'   keep_nas = TRUE
 #' )
 #'
+#' # Example 1c: Using option `keep_nas = exprs(ADTF)` to derive MAP in the case where
+#' # some/all values of a variable used in the computation are missing but ignoring ADTF
+#'
+#' derive_param_computed(
+#'   advs,
+#'   by_vars = exprs(USUBJID, VISIT),
+#'   parameters = c("SYSBP", "DIABP"),
+#'   set_values_to = exprs(
+#'     AVAL = (AVAL.SYSBP + 2 * AVAL.DIABP) / 3,
+#'     PARAMCD = "MAP",
+#'     PARAM = "Mean Arterial Pressure (mmHg)",
+#'     AVALU = "mmHg",
+#'     ADT = ADT.SYSBP,
+#'     ADTF = ADTF.SYSBP
```

Comment:
Unfortunately, the ADaM IG is not very clear regarding imputation flags for derived records. Usually one date of the contributing records is selected as the analysis date for the new record. I would consider both retaining the imputation flags and not retaining them as acceptable.
Setting `ADTF = "Y"` for all records looks strange and confusing to me. I think the imputation flags should indicate missing information but not that a date was derived.
However, I would leave the example as it is because it shows how the function work.

Reply:
Ok, nothing critical here. Agree to leave as is. Thanks for your thoughts.